packages/next/src/next-devtools/dev-overlay/hooks/use-shortcuts.ts

```diff
@@ -35,8 +35,7 @@ export function useShortcuts(
 
     window.addEventListener('keydown', handleKeyDown)
     return () => window.removeEventListener('keydown', handleKeyDown)
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [shortcuts])
+  }, [rootRef, shortcuts])
```

Comment:
Using a ref here is fine since you only read in an event handler. Would be wasteful to re-attach the handler just so that it closes over the right value.